src/index.js

```diff
@@ -89,12 +114,26 @@ app.get('/poll', (req, res) => {
   res.send({log: rollLog})
 })
 
+app.post('/clearLog', (req, res) => {
+  req = clearLog()
```

Comment:
I don't think we want to reassign req here

Reply:
```suggestion
clearLog()
```

Reply:
i would do add a 
```
 res.send({log: rollLog})
```
then you could drop the `;poll()` from l98

Reply:
My bad on the req... and yes could do this, however is there any need to return the rollLog? Calling poll() on FE seems like a neater implementation.

Reply:
yeah i guess  `res.send({log: rollLog})` is kinda pointless - it will be []. 
Maybe just respond with 200,  and on the FE call display([]) to update the dom. 

i don't think we really need to poll again, it happens every 2s anyway..